sklearn/dummy.py

```diff
@@ -190,8 +191,8 @@ def predict(self, X):
 
         Parameters
         ----------
-        X : {array-like, object with finite length or shape}
-            Training data, requires length = n_samples
+        X : array-like of shape (n_samples, n_features)
+            Test data.
```

Comment:
Predict receives test data.

Reply:
good catch
I guess we have a bunch of these throughout the codebase

Reply:
IIRC, I have seen this several times across the documentation (maybe it can be a good issue for the sprint?).